testing/regress/hpcc/util/ecl/command.py

```diff
@@ -78,7 +78,9 @@ def runCmd(self, cmd, cluster, eclfile, report, **kwargs):
 
             args = args + eclfile.getStoredInputParameters()
 
-            args.append('-XTargetIP=' + server)
+            if '-XTargetIP' not in "".join(args):
+                logging.debug("%3d. results:'%s'", eclfile.getTaskId(),  "'-XTargetIP' is not in args")
+                args.append('-XTargetIP=' + server)
```

Comment:
This checks if '-XTargetIP' exist in the generated command line. If not then add it with default server IP value. If I know well it is used by 2 test cases (soapcall-multihttpheader.ecl and httpcall_multiheader.ecl)  in Regression Suite.